src/pools/composableStable/composableStablePool.ts

```diff
@@ -233,31 +213,27 @@ export class ComposableStablePool extends PhantomStablePool {
      */
     _calcBptOutGivenExactTokensIn(amountsIn: BigNumber[]): BigNumber {
         try {
-            const amountsInHuman = amountsIn.map((amount, i) =>
-                formatFixed(amount, this.tokens[i].decimals)
+            const amountsInNormalised = new Array(amountsIn.length).fill(
+                BigInt(0)
             );
-            const amountsInWithRate = new Array(amountsIn.length).fill(
+            const balancesNormalised = new Array(amountsIn.length).fill(
                 BigInt(0)
             );
-            const balancesEvm = new Array(amountsIn.length).fill(BigInt(0));
             // token balances are stored in human scale and must be EVM for maths
```

Comment:
Would it be good to update this comment to explain normalized rather than EVM. Similar to function above.